README.md

```diff
@@ -165,9 +172,21 @@ const urlPrefix = server.app.config.get('urlPrefix');
 // crumb WILL use secure cookies.
 ```
 
+You can also easily determine whether a plugin should be enabled at all. See the code below for an example disabling `hapi-pino` in the `manifest.json` file.
+
+```javascript
+{
+    "register": {
+        "hapi-pino": {
+            "enabled": false
```

Comment:
This is just disabling the plugin. I was thinking you'd use filters or protocols to do this:

**confidence**:
```
"enabled": {
    "$filter": "env.NODE_ENV",
    "$default": false,
    "development": true
}
```

**shortstop:**
```
"enabled": "env:CRUMB_ENABLED|b"
```

We could just take the sentence out too, or rephrase it so it doesn't use "determine".